docs/source/azure/multiplesubs.rst

```diff
@@ -0,0 +1,18 @@
+.. _azure_multiplesubs:
+
+Multiple Subscriptions
+======================
+
+See 
+`C7N-Org Readme <https://github.com/capitalone/cloud-custodian/tree/master/tools/c7n_org>`_ 
+for general information on running Cloud Custodian across accounts and subscriptions.
+
+If you're using an Azure Service Principal for executing c7n-org
+you'll need to ensure that the principal has access to multiple
+subscriptions. For instructions on creating a service principal and granting access
+across subscriptions, visit the `Azure authentication docs
+page <http://capitalone.github.io/cloud-custodian/docs/azure/authentication.html>`_
+
+**Note**: There are pending issues with running Cloud Custodian on Windows & MacOS 
```

Comment:
fwiw, the macos issue is around very specific versions of osx (high sierra) using a particular version of python (2.7)

i've filed an issue with additional details against python https://bugs.python.org/issue33725